workflows/servicenow/versions/1.0.0/README.md

```diff
@@ -0,0 +1,23 @@
+# ServiceNow
+
+## Summary
+
+A set of templates to perform operations against a ServiceNow instance
+
+## Requirements
+1. create a secret containing 2 pieces of informations to connect to your
+ServiceNow instance. See [example](../../assets/sn_auth.yaml)
+  - username
+  - password
+
+## Templates
+1. [createcr](./docs/createcr.md)
+1. [updatecr](./docs/updatecr.md)
+1. [closecr](./docs/closecr.md)
```

Comment:
you need to put the actual url- for example
 https://github.com/codefresh-io/argo-hub/blob/main/workflows/servicenow/versions/1.0.0/docs/closecr.md

Reply:
No, this is a relative path. It works fine